src/installer/installer.go

```diff
@@ -355,3 +366,56 @@ func (i *installer) cleanupInstallDevice() error {
 
 	return i.ops.RemovePV(i.Device)
 }
+
+func (i *installer) verifyHostCanMoveToConfigurationStatus(inventoryHostsMapWithIp *map[string][]string) {
```

Comment:
why do you need to pass the map by pointer, and then de-reference it? map is a reference anyway , and deleting while iterating is also safe